packages/datadog-instrumentations/src/helpers/instrument.js

```diff
@@ -24,23 +43,19 @@ exports.tracingChannel = function (name) {
 
 /**
  * @param {object} args
- * @param {string|string[]} args.name module name
- * @param {string[]} args.versions array of semver range strings
+ * @param {string} args.name module name
```

Comment:
why remove the possibility to have multiple modules on the same hook ?

Reply:
This is not how it worked. It created multiple hook entries, each with an individual name but the same hook method.
That was to overcome handling the `node:` prefix issue which is now automatically handled in all cases.